awx/api/serializers.py

```diff
@@ -43,6 +43,8 @@
 # Django-Polymorphic
 from polymorphic.models import PolymorphicModel
 
+from ansible_base.utils.models import get_type_for_model
```

Comment:
This "type" notion was always a very specific AWX-ism. I guess it probably works for other apps. I think the gist is wanting "job_template" as opposed to the standard model syntax of "jobtemplate".